ibm_catalog.json

```diff
@@ -247,11 +222,11 @@
               "key": "key_type",
               "options": [
                 {
-                  "displayname": "rsa",
+                  "displayname": "Rsa",
```

Comment:
Didn't found `key_bits` anywhere in the `variables.tf` nor in `ibm_catalog.json`, so how will the algorithm be selected as `RSA 2048` or `RSA 4096`, same applies to EC algorithm.

<img width="348" height="221" alt="image" src="https://github.com/user-attachments/assets/d4ac87f7-90f4-4f2c-8129-35232f433e96" />

[key bits](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/sm_private_certificate_configuration_root_ca#key_bits-2) is not part of code and should be included in the code. Please create another issue to handle this.